my_pyscf/mcscf/lassi.py

```diff
@@ -577,5 +577,10 @@ def root_make_rdm12s (las, ci, si, state=0, orbsym=None, soc=None, break_symmetr
                                       break_symmetry=break_symmetry, opt=opt)
     return rdm1s[0], rdm2s[0]
 
-
-
+def root_make_casdm12(las, ci, si, state=0, orbsym=None, soc=None, break_symmetry=None, opt=1):
+    verbosecopy = las.verbose
```

Comment:
```
with lib.temporary_env (las, verbose=2):
    casdm1, casdm2s = root_make_rdm12s (... # in indented block underneath "with" line
```
is safer. it'll never fail to put the original las.verbose back, even if an exception is raised.